File: VB/DragDropColumns/MyColumnChooserDropTarget/MyColumnChooserDropTargetFactory.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.Grid
Imports DevExpress.Xpf.Core
Imports System.Windows

Namespace DragDropColumns
    Public Class MyColumnChooserDropTargetFactory
        Inherits GridDropTargetFactoryBase

        Protected Overrides Function CreateDropTarget(ByVal dropTargetElement As UIElement) As IDropTarget
            Return New MyColumnChooserDropTarget()
        End Function
    End Class
End Namespace

File: CS/DragDropColumns/MyColumnChooserDropTarget/MyColumnChooserDropTargetFactory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.Grid;
using DevExpress.Xpf.Core;
using System.Windows;

namespace DragDropColumns
{
    public class MyColumnChooserDropTargetFactory : GridDropTargetFactoryBase
    {
        protected override IDropTarget CreateDropTarget(UIElement dropTargetElement)
        {
            return new MyColumnChooserDropTarget();
        }
    }
}
